DependencyInjection/FOSElasticaExtension.php

```diff
@@ -110,6 +110,11 @@ private function loadClients(array $clients, ContainerBuilder $container)
             if (false !== $logger) {
                 $clientDef->addMethodCall('setLogger', array(new Reference($logger)));
             }
+
+            if (isset($clientConfig['connections'][0]['connectTimeout'])) {
```

Comment:
This code was actually not needed. I dug a bit deeper and found that if a config value of connectTimeout is given, that does the trick already and it should set everything correctly inside the underlying elastica library.